ios/FluentUI/Vnext/List/List.swift

```diff
@@ -70,12 +70,32 @@ public struct MSFListView: View {
                           with: windowProvider)
     }
 
+    private func findLastCell(_ lastCell: MSFListCellState) -> MSFListCellState {
+        if let children = lastCell.children {
+            return lastDivider(findLastCell(children[children.count - 1]))
+        } else {
+            return lastDivider(lastCell)
+        }
+    }
+
+    private func lastDivider(_ cell: MSFListCellState) -> MSFListCellState {
+        let state = cell
+        state.hasDivider = false
+        return state
+    }
+
     private func updateCellDividers() -> [MSFListSectionState] {
         state.sections.forEach { section in
             section.cells.forEach { cell in
                 cell.hasDivider = section.hasDividers
             }
-            section.cells.last?.hasDivider = false
+            if let last = section.cells.last {
```

Comment:
how you are organizing the code can be better here. while you are enumerating (recursively or not) the lambda block should be the one setting the state.hasDivider  to false or true depending on it is a last cell.  how it is currently organized doesn't mix and match how you are traversing the tree and changing its values